docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -146,6 +146,14 @@ To set a different default mode, pass the `defaultMode` prop to the ThemeProvide
 The `defaultMode` value can be `'light'`, `'dark'`, or `'system'`.
 :::
 
+### InitColorSchemeScript component
```

Comment:
Should we revert the docs order? It looks like if `InitColorSchemeScript` is used, `<ThemeProvider theme={theme} defaultMode="dark">` is a noop.